syntax				= "proto3";
option optimize_for = CODE_SIZE;

message Cfg_ItemFormula
{

		uint32	id				= 1; // id
		uint32	sort			= 2; //合成类型
		uint32	target_itemtype = 3; //合成出来的物品类型ID
		uint32	target_itemnum	= 4; //合成出来的物品数量
		uint32	target_flag		= 5; //合成出来的物品掩码
		uint32	lev_req			= 6; //等级需求
		float	succ_rate		= 7; //成功率
		uint32	rumor			= 8; //传闻编号
		message ReqData
		{
			uint32 req_type = 1; //需求
			uint32 req_num	= 2;
		};

		repeated ReqData req_data_list = 10;

}